ee/frontend/mobile-replay/mobile.types.ts

```diff
@@ -122,6 +122,60 @@ type wireframeBase = {
     style?: MobileStyles
 }
 
+export type wireframeInputBase = wireframeBase & {
+    type: 'input'
+    disabled: boolean
+}
+
+export type wireframeCheckBox = wireframeInputBase & {
+    inputType: 'checkbox'
+    checked: boolean
+    label?: string
+}
+
+export type wireframeRadioGroup = wireframeBase & {
+    groupName: string
```

Comment:
```suggestion
    type: 'radio group'
    groupName?: string
```

@marandaneto is there any value to the App to specify this group name? Otherwise I can pass a UUID to all children radios...

Reply:
ok, let's leave it out... easier to add than to remove 👍